guide/creating-your-bot/main-file.md

```diff
@@ -17,9 +17,11 @@ const { token } = require('./config.json');
 const client = new Client({ intents: [GatewayIntentBits.Guilds] });
 
 // When the client is ready, run this code (only once)
-// We use 'c' for the event parameter to keep it separate from the already defined 'client'
-client.once(Events.ClientReady, c => {
-	console.log(`Ready! Logged in as ${c.user.tag}`);
+// We use 'readyClient' for the event parameter to keep it separate from the already defined 'client'
+// This is important for TypeScript developers as the parameter will be a client that is ready
+// with some properties being non-nullable
```

Comment:
```suggestion
// The distinction between `client: Client<boolean>` and `readyClient: Client<true>` is important for TypeScript Developers.
// It makes some properties non-nullable.
```
I think i prefer more concise wording here, while adding some verbosity about the specific types.